packages/elements/src/components/ui/action-menu/ActionMenuItem.tsx

```diff
@@ -49,7 +49,11 @@ export const ActionMenuItem = forwardRef<
   return (
     <button
       {...props}
-      className={cx(styles.actionMenuItem, styles[variant], className)}
+      className={cx(
+        styles.actionMenuItem,
+        styles[props.disabled ? "disabled" : variant],
```

Comment:
This is still an issue. There is no `.disabled` class in the css so this is the same as writing `styles[props.disabled ? undefined : variant]`. I think we should keep the variant rather than throwing it away if the button is disabled. Instead we could solve it as suggested, by moving `:disabled` in the css file so that it gains priority over the variant.

```suggestion
        styles[variant],
```